src/confighttp.cpp

```diff
@@ -488,6 +488,45 @@ void uploadCover(resp_https_t response, req_https_t request) {
   outputTree.put("path", path);
 }
 
+void savePrepCmd(resp_https_t response, req_https_t request) {
```

Comment:
Unfortunately this is probably not an easy change, but... instead of adding a new endpoint and function... can we just use the saveConfig endpoint and attach them to the config object?

Reply:
Might be a bit of a challenge since sunshine.conf isn't JSON, but I should be able to make it work. Would we also want to move the global prep tab onto the config page then? I feel like that makes more sense anyway.

Reply:
> Would we also want to move the global prep tab onto the config page then?

Yea, I think it makes the most sense.

There's a couple of options that are like a json list (resolution and fps I believe), just going from memory, so that might not be 100% accurate.

Reply:
Now stored in config and setup moved to the config page.
![image](https://user-images.githubusercontent.com/3165068/224227855-17b620fb-5f21-473c-baa2-ce85a03ce064.png)